### PR TITLE
Add gradle configuration for static code analyzer

### PR DESCRIPTION
**Pull request by f2rf2r**

### CODE REVIEW

**Comment by prfarlow1** [comment]
A couple observations:
CompareObjectsWithEquals seems helpful but it does flag View comparison using ==
I'm not a fan of the UselessParentheses rule; I think sometimes they add clarity

File: app/static-analysis.gradle [hunk 1]
```diff
@@ -8,6 +8,8 @@ task pmd(type: Pmd) {
     include '**/*.java'
     exclude '**/gen/**'
 
+
+
```
Comment (prfarlow1) [inline app/static-analysis.gradle]: extra whitespace

**Comment by gsintrepid** [comment]
I agree with @prfarlow1, in Stetson there were 23 "UselessParentheses" warnings and many were useful for several/long comparisons.
The other warnings made sense!

**Comment by deanshi** [comment]
For the PMD, is there a reason why we're not using the Empty Code ruleset?

**Comment by f2rf2r** [comment]
@deanshi I just forgot/overlooked it. I have added that in the gradle plugin pr. I also changed this project to use the plugin now, so future comments regarding the rulesets should move to that repo. https://github.com/IntrepidPursuits/static-analysis-gradle-plugin/pull/1

**Review by deanshi — APPROVED** [review]
Looks good to me :shipit:

File: app/build.gradle [hunk 2]
```diff
@@ -1,3 +1,7 @@
+plugins {
+    id "io.intrepid.static-analysis" version "1.0-alpha6"
```
Comment (deanshi) [inline app/build.gradle]: Will this be updated to 1.0 to match the README of the `static-analysis-gradle-plugin` once it's merged in?

Comment (f2rf2r) [reply app/build.gradle]: yes

**Pull request merged**